web/public/locales/de.json

```
@@ -44,7 +44,11 @@
         "noParserInfo": "Wir haben noch keine Daten für diese Zone.<br/> Willst du uns helfen, das zu ändern? Wirke mit, indem du <a href=\"%s\" target=\"_blank\">Datenquellen hinzufügst</a>.",
         "now": "Jetzt",
         "electricity": "Elektrizität",
-        "dataIsDelayed": "Die Daten in dieser Region können bis zu %s Stunden verzögert sein. Das Darstellen von Live-Daten steht nicht zur Verfügung."
+        "dataIsDelayed": "Die Daten in dieser Region können bis zu %s Stunden verzögert sein. Das Darstellen von Live-Daten steht nicht zur Verfügung.",
+        "estimated": "Geschätzt",
+        "dataIsEstimated": "Teile der Daten für diese Region sind geschätzt. Mehr über unsere Schätzungsmethoden kannst du <a href=\"https://github.com/electricityMap/electricitymap-contrib/wiki/Estimation-methods\" target=\"_blank\">hier</a> lesen.",
+        "estimatedTooltip": "Die Echtzeit Daten für diese Zone sind auf Basis der Daten der Quelle und den Annahmen des Modells geschätzt.",
```

I'm not a native speaker, but I think this might work better with a dash in Echtzeit-Daten?

```suggestion
        "estimatedTooltip": "Die Echtzeit-Daten für diese Zone sind auf Basis der Daten der Quelle und den Annahmen des Modells geschätzt.",
```

---

yes